routes/folders.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const router = express.Router();
+
+// Import middleware
+const { attachRouteHandlerWrapper } = require('../middleware/routeHandler')
+
+// Import classes 
+const { Directory, FolderType } = require('../classes/Directory.js');
+
+// List pages and directories in folder
+async function listFolderContent (req, res, next) {
+    const { accessToken } = req
+    const { siteName } = req.params
+    const { path } = req.query // paths begin with _
```

Comment:
nit: from our previous discussions, we're using this route to access both collections and nested folders (can also be generalised to other image, resource folders?), so the comment here `// paths begin with _` is misleading

Reply:
fixed with [c6f99d1](https://github.com/isomerpages/isomercms-backend/pull/108/commits/83f314d0dde7095a6ebb321679a5d0c719a3d0d4)